mmdet/core/anchor/anchor_generator.py

```diff
@@ -10,14 +11,16 @@ class AnchorGenerator(object):
     """Standard anchor generator for 2D anchor-based detectors
 
     Args:
-        strides (list[int]): Strides of anchors in multiple feture levels.
+        strides (list[int] | list[tuple[int, int]]): Strides of anchors
+            in multiple feture levels.
         ratios (list[float]): The list of ratios between the height and width
             of anchors in a single level.
         scales (list[int] | None): Anchor scales for anchors in a single level.
             It cannot be set at the same time if `octave_base_scale` and
             `scales_per_octave` are set.
-        base_sizes (list[int] | None): The basic sizes of anchors in multiple
-            levels. If None is given, strides will be used as base_sizes.
+        base_sizes (list[int] | list[tuple[int, int]] | None): The basic sizes
+            of anchors in multiple levels.
+            If None is given, strides will be used as base_sizes.
```

Comment:
Explain in the documentation that how strides are used as base_sizes (especially when it is a list of tuple).